.github/workflows/scripts/biome_disable_check.py

```
@@ -0,0 +1,160 @@
+
+def has_biome_ignore(file_path: str) -> bool:
+    """Check if a file contains biome-ignore statements.
+
+    Args:
+        file_path: Path to the file.
+
+    Returns:
+        bool: True if biome-ignore statement is found, False otherwise.
+    """
+    biome_ignore_pattern = re.compile(
+        r"//\s*biome-ignore.*$", re.IGNORECASE | re.MULTILINE
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)